libnopegl/src/backends/gl/glcontext.c

```diff
@@ -402,6 +407,43 @@ static int glcontext_probe_settings(struct glcontext *glcontext)
     return 0;
 }
 
+static int glcontext_check_driver(struct glcontext *glcontext)
+{
+    const char *gl_version = (const char *)ngli_glGetString(glcontext, GL_VERSION);
+    if (!gl_version) {
+        LOG(ERROR, "could not get OpenGL version");
+        return NGL_ERROR_BUG;
+    }
+
+    int mesa_version[3] = {0};
+    const char *mesa = strstr(gl_version, "Mesa");
+    if (mesa) {
+        int ret = sscanf(mesa,
+                         "Mesa %d.%d.%d",
+                         &mesa_version[0],
+                         &mesa_version[1],
+                         &mesa_version[2]);
+        if (ret != 3) {
+            LOG(ERROR, "could not parse Mesa version (%s)", mesa);
```

Comment:
nit: `"could not parse Mesa version: \"%s\""`

Reply:
Done.